### DISCUSSION TITLE
Routing IPOnDemand not working

### DISCUSSION THREAD

**Discussion by Maneeha1234SL** [v2fly/v2ray-core#1633] [Q&A]
![image](https://user-images.githubusercontent.com/83775004/155836496-f0255a2c-5420-4263-882b-717df1f94ec0.png)
![image](https://user-images.githubusercontent.com/83775004/155836500-ab5e3698-4105-4749-a30a-169dde442a49.png)

  
Please solve this I need to play multiplayer in this game

Game-Asphalt 9: Legends 9 (Microsoft Store Game) { https://www.microsoft.com/en-ws/p/asphalt-9-legends/9nzqpt0mwtd0?activetab=pivot:overviewtab }

If this is converted to an IP then multiplayer mode can be played in this game.
V2Ray 4.44.0
xray-core v1.5.3

The ip type associated with the problem = ip.a9.gold0010.gameloft.com

**Comment by kslr** [comment]
use wireguard or netch
